spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -581,6 +583,12 @@ def copy(self):
         clipboard = QApplication.clipboard()
         clipboard.setText(cliptxt)
 
+    def edit_item(self):
+        """Edit item"""
+        index = self.currentIndex()
+        if index.isValid():
+            self.edit(index)
+
 
 class ArrayEditorWidget(QWidget):
```

Comment:
I made the widget inherit from `SpyderWidgetMixin` but I don't think I understand how to use its API to build the toolbar. I saw the `.create_toolbar()` function which looks promising from its name but it creates a `QToolBar` instead of a `SpyderToolbar`.

Reply:
Right, that's something we need to think better for that mixin. Perhaps adding `create_spyder_toolbar` and renaming `create_toolbar` to `create_qtoolbar` would work, but that's not something for this PR.